### ISSUE TITLE
[REQ] Shuffle All Button

### ISSUE THREAD

**Issue by kavbrian21** [CappielloAntonio/tempo#12]
It would be great if we could have a button to shuffle all tracks in the library.

In case it's useful, here is a [a thread with the Navidrome developer discussing possible solutions for implementing this in a scalable way](https://github.com/navidrome/navidrome/issues/256#issuecomment-621967871).

**Comment by CappielloAntonio** [comment]
Hi, I'll start by telling you that I've added a shuffling button to the app. By pressing the button, as you can guess, the random playback of your music library starts.

But there is a but. There are two reasons why I didn't include this button from the beginning:

- In the case of medium/large music libraries, the results would be extremely random, one would go from different genres, to different decades, to different moods. For this reason I created the "Made for you", "Radio Station" and "Best of" sections which, as you can imagine, reproduce music more or less randomly starting from a track, an album or an artist.

- Unless I make multiple consecutive calls to the endpoint set up for obtaining random songs, the limit of tracks I can retrieve will be 500 per call. Unlike a server application that can overcome this limit by contacting the db directly, I have to "talk" through the [Subsonic API](http://www.subsonic.org/pages/api.jsp#getRandomSongs).

tl;dr I added the button, limiting the number of tracks I retrieve so as not to slow down the app and avoiding making subsequent calls for the same reason.
